### PR TITLE
[Docker] Phase 1: Splitting scraper from API

### LINKED ISSUE #594: Error fetching
JHU File not found at this URL https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_daily_reports/04-24-2020.csv

This happens because CSV is not generated yet for that day. Can this error be handled more gracefully?

Error
```
{
  message: '[2020-04-25T06:46:37.768Z]: Error: Requesting JHULocations failed!',
  error: 'Request failed with status code 404',
  stack: 'Error: Request failed with status code 404\n' +
    '    at createError (/Users/skavinvarnan/Projects/Node/NovelAPI/node_modules/axios/lib/core/createError.js:16:15)\n' +
    '    at settle (/Users/skavinvarnan/Projects/Node/NovelAPI/node_modules/axios/lib/core/settle.js:17:12)\n' +
    '    at IncomingMessage.handleStreamEnd (/Users/skavinvarnan/Projects/Node/NovelAPI/node_modules/axios/lib/adapters/http.js:236:11)\n' +
    '    at IncomingMessage.emit (events.js:215:7)\n' +
    '    at endReadableNT (_stream_readable.js:1183:12)\n' +
    '    at processTicksAndRejections (internal/process/task_queues.js:80:21)'
}
(node:91365) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'data' of undefined
    at Object.jhudataV2 (/NovelAPI/scrapers/jhuLocations.js:29:25)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

```

### PR DESCRIPTION
**Pull request by buster95**
# In this PR

- Splitting Scraper from API
- Others issues when there isn't internet connection
- Correct timezone for github actions based on JHU repo GMT-4

### Issues Related
closes #594 

### Important
* **With this PR you need to move config.json file into config folder**

This PR is one of many for full Docker integration on the server and is a compilation of changes from PR #517 thanks @sam-perez

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ node_modules/
 json.sqlite
 .vscode/
 config.json
+/config/config.json
```
Comment (buster95) [inline .gitignore:5]: good @ebwinters I forgot this

**Comment by ebwinters** [comment]
@Nekiono before we merge this, you need to change `config.json` location to `/config/config.json` on server

**Comment by puf17640** [comment]
closes #594

**Comment by Nekiono** [comment]
> @Nekiono before we merge this, you need to copy `config.json` to a new location `/config/config.json` on server

this shall be symlinked

**Pull request merged**